eth-bytecode-db/eth-bytecode-db/tests/verification_test_helpers/mod.rs

```diff
@@ -15,33 +16,61 @@ use eth_bytecode_db::verification::{
 use pretty_assertions::assert_eq;
 use sea_orm::{DatabaseConnection, EntityTrait};
 use smart_contract_verifier_proto::blockscout::smart_contract_verifier::v1::VerifyResponse;
-use smart_contract_veriifer_mock::{MockSolidityVerifierService, MockVyperVerifierService};
+use smart_contract_veriifer_mock::{
+    MockSolidityVerifierService, MockSourcifyVerifierService, MockVyperVerifierService,
+    SmartContractVerifierServer,
+};
 use std::{collections::HashSet, future::Future, str::FromStr, sync::Arc};
 use test_input_data::TestInputData;
 use tonic::transport::Uri;
 
 type AddIntoServiceFn<Service, GrpcT> = Arc<dyn Fn(&mut Service, GrpcT, VerifyResponse)>;
 
 #[derive(Clone)]
-pub enum VerifierServiceType<GrpcT> {
+pub enum VerifierServiceType<Request, GrpcT>
+// where GrpcT: From<Request>
+{
     Solidity {
```

Comment:
why not make this as trait, with `type Request;`, and then have here just 3 `Arc<dyn Trait<Request = Mock..>>`?

Reply:
Not exactly `type Request`, but updated to use a trait instead of enum